typescript/web/src/components/dataset-classes/upsert-class-modal/create-label-class.mutation.ts

```diff
@@ -12,9 +12,9 @@ import { useCallback } from "react";
 import { v4 as uuid } from "uuid";
 import { DATASET_LABEL_CLASSES_QUERY } from "./dataset-label-classes.query";
 
-export const CREATE_LABEL_CLASS_MUTATION = gql`
+export const createLabelClassMutation = gql`
```

Comment:
We agreed to use `CONSTANT_CASE` (to denote that these globals should be considered as _deeply-immutable_). Why did you do the inverse?

Reply:
It's a rebase conflict that I resolved. It's specifically designed to annoy you

Reply:
Ahah! Of course it's not 😅
I now understand why and I agree that it was easier to resolve the conflict this way 👍 
Sorry for asking such questions while it was still a draft.